config.go

```diff
@@ -0,0 +1,38 @@
+package db
```

Comment:
I added this because we're starting to require some configuration beyond the basics, this replaces `db.Debug` and adds support for a logger.